src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -96,21 +100,56 @@ export default function AdaptiveCard({
       };
     });
   };
+
   const setInput = useCallback((input) => {
     setInputs((prevInputs) => ({...prevInputs, [input.id]: input}));
   }, [setInputs]);
-  const getValue = (id, defval = '') => ((id in inputs && inputs[id].value) ? inputs[id].value : defval);
-  const getAllValues = () => Object.entries(inputs).reduce((allValues, [id, input]) => (
-    {...allValues, [id]: input.value}
-  ), {});
+
+  const getValue = (id, defval = '') => ((id in inputs && inputs[id].value !== undefined) ? inputs[id].value : defval);
+
+  const getAllValues = () => Object.fromEntries(Object.entries(inputs).map(
```

Comment:
you can simplify this using the util function `mapValues` described in a comment below